code/game/objects/items/stacks/packagewrap.dm

```diff
@@ -134,10 +134,10 @@
 /obj/item/delivery/attack_self(mob/user as mob)
 	if(wrapped)
 		if(ishuman(user))
+			qdel(src)
```

Comment:
Since the Destroy moves wrapped onto the tile, could this not cause the user to drop the wrapped item?